yaml/water.yaml

```diff
@@ -18,6 +18,27 @@ globals:
           - when:
               intermittent: 'yes'
             then: true
+  - &water_polygon_min_zoom
+    lookup:
+      key: { col: way_area }
+      op: '>='
+      table:
+        # limit show smaller landuse areas to higher zooms
+        - [ 0, 40000000000 ]
+        - [ 2, 20000000000 ]
+        # some weird natural earth scale set transition
+        - [ 3, 80000000000 ]
+        - [ 4,  5000000000 ]
+        - [ 5,   700000000 ]
+        - [ 6,   500000000 ]
+        - [ 7,   160000000 ]
```

Comment:
Suggest removing lines for zooms 0-7 as those are from Natural Earth and should be taken from the data, including the junk comment copy-pasted from Walkabout for `# limit show smaller landuse areas to higher zooms`.

Reply:
(Should add comment to that effect... &/or keep filters as backstop against future OSM / NE zoom transition changes?)

Reply:
Sounds good. Clipped to 7 and quickly tested locally. Looks okay to me. Fixed in 67aa978edf2298a6028cdbab8ab762aaf2f59f6a.